build.gradle

```
@@ -4,23 +4,58 @@ plugins {
     id 'java'
 }
 
-group = 'camp.nextstep.edu'
-version = '0.0.1-SNAPSHOT'
-sourceCompatibility = '1.8'
+allprojects {
+    group = 'camp.nextstep.edu'
+    version = '0.0.1-SNAPSHOT'
+    sourceCompatibility = '1.8'
 
-repositories {
-    mavenCentral()
+    apply plugin: 'org.springframework.boot'
+    apply plugin: 'io.spring.dependency-management'
+    apply plugin: 'java'
+
+    repositories {
+        mavenCentral()
+    }
+
+    dependencies {
+        implementation 'org.springframework.boot:spring-boot-starter-actuator'
+        //jpa
+        implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+        implementation 'org.springframework.boot:spring-boot-starter-web'
+        implementation 'org.flywaydb:flyway-core'
+        runtimeOnly 'com.h2database:h2'
+        testImplementation 'org.springframework.boot:spring-boot-starter-test'
+    }
 }
 
 dependencies {
-    implementation 'org.springframework.boot:spring-boot-starter-actuator'
-    implementation 'org.springframework.boot:spring-boot-starter-jdbc'
-    //jpa
-    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
-    implementation 'org.springframework.boot:spring-boot-starter-web'
-    implementation 'org.flywaydb:flyway-core'
-    runtimeOnly 'com.h2database:h2'
-    testImplementation 'org.springframework.boot:spring-boot-starter-test'
+    implementation project(':module-product')
+    implementation project(':module-menugroup')
+    implementation project(':module-menu')
+    implementation project(':module-table')
+    implementation project(':module-tablegroup')
+    implementation project(':module-order')
+}
+
+project(':module-menu') {
+    dependencies {
+        implementation project(':module-product')
+        implementation project(':module-menugroup')
+    }
+}
+
+project(':module-tablegroup') {
+    dependencies {
+        implementation project(':module-table')
+    }
+}
+
+project(':module-order') {
+    dependencies {
+        implementation project(':module-menu')
+        implementation project(':module-table')
+        implementation project(':module-tablegroup')
+    }
 }
```

다음과 같이 해주면 제대로 동작이 되네요! 😄 
```suggestion
project(':module-order') {
    dependencies {
        implementation project(':module-menu')
        implementation project(':module-table')
        implementation project(':module-tablegroup')
        testCompile project(':module-util').sourceSets.test.output
    }
}
```